org.eclipse.text.quicksearch/src/org/eclipse/text/quicksearch/internal/core/QuickTextSearchRequestor.java

```diff
@@ -24,30 +24,31 @@
  *
  * @author Kris De Volder
  */
-public class QuickTextSearchRequestor {
+@SuppressWarnings("unused")
+public interface QuickTextSearchRequestor {
```

Comment:
Shouldn't this be **renamed** too following Eclipse conventions to `IQuickTextSearchRequestor` ?

Reply:
@kthoms Would you please adjust according to @iloveeclipse request?